frontend/src/app/components/datasets/create-dataset/create-dataset.component.html

```diff
@@ -11,7 +11,7 @@ <h2>Basic information</h2>
             <mat-label>Name</mat-label>
             <input matInput placeholder="Name" formControlName="name">
             <mat-error *ngIf="datasetForm.controls['name'].hasError('nameValidation')">
-                Huggingface dataset name can only contain alphanumerics, "_", "-" and "."
+                Huggingface dataset name can only contain alphanumerics and "_".
```

Comment:
This is not in line with `validators.ts`

Reply:
it should be the same since these are the only allowed characters `[a-zA-Z0-9_]`